src/Microsoft.ML.StandardLearners/Standard/Online/AveragedPerceptron.cs

```diff
@@ -126,23 +126,23 @@ internal AveragedPerceptronTrainer(IHostEnvironment env, Options options)
         /// <param name="featureColumnName">The name of the feature column.</param>
         /// <param name="learningRate">The learning rate. </param>
         /// <param name="decreaseLearningRate">Whether to decrease learning rate as iterations progress.</param>
-        /// <param name="l2RegularizerWeight">L2 Regularization Weight.</param>
+        /// <param name="l2Regularization">Weight of L2 regularization term.</param>
         /// <param name="numIterations">The number of training iterations.</param>
```

Comment:
```suggestion
        /// <param name="maximumNumberOfIterations">The number of training iterations.</param>
```

Reply:
I don't get your suggestion.

---
In reply to: [263972642](https://github.com/dotnet/machinelearning/pull/2892#discussion_r263972642) [](ancestors = 263972642)